go.mod

```diff
@@ -81,7 +81,7 @@ require (
 	github.com/containers/storage v1.46.0 // indirect
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
 	github.com/docker/distribution v2.8.2+incompatible // indirect
-	github.com/docker/docker v23.0.2+incompatible // indirect
+	github.com/docker/docker v25.0.5+incompatible // indirect
```

Comment:
It's possible that bumping our container images library `github.com/containers/image/v5` to something newer would automatically bump the runc/docker ones as well.
I see cri-o main uses `v5.30.0` - https://github.com/cri-o/cri-o/pull/8133

Reply:
I tried that and it didnt bump it to new enough version without the CVE..
Also I didnt see we have cri-o in our go.mod..

Reply:
hmm now I see it should be the same version, let me try again.

Reply:
Right I see, makes sense